src/utils/I18nUtils.ts

```diff
@@ -0,0 +1,9 @@
+const pluralRules = new Intl.PluralRules("en-US");
+
+export function pluralize(count: number, singular: string, plural?: string, skipCountInReturnedValue = false): string {
+  const pluralRule = pluralRules.select(count);
+  if (pluralRule === "one") {
+    return `${!skipCountInReturnedValue ? `${count} ` : ""}${singular}`;
```

Comment:
I think these would be easier to follow if you refactor the if skipCount out earlier, and then you could add the resulting prefix unconditionally.